WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -147,10 +147,8 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         ((WordPress) getActivity().getApplicationContext()).component().inject(this);
         mDispatcher.register(this);
+        PreferenceManager.setDefaultValues(getActivity(), R.xml.account_settings, false);
 
-        if (getActivity() != null) {
```

Comment:
The reason I removed this check is that, if the `getActivity` is null, it'll never make it to this line due to 148 and I think we want the app to crash if there is an NPE there.